### PR TITLE
Finish join group & Remove 'Monthly' button & Fix radio button style

### PR DESCRIPTION
**Pull request by JipengSun**
## Trello Ticket :ticket: ##
https://trello.com/c/vVMl8YIN/66-be-make-joingroup-page-functional-responsive
https://trello.com/c/UXmGaVoy/67-look-for-radio-buttons-edit-to-allow-click-on-text
https://trello.com/c/5Tzx5HQb/77-be-remove-monthly-habit-choice

## Summary :chart_with_upwards_trend: ##
Finish join the group and delete 'monthly' button and fix the radio button style. 

## Test plan :clipboard: ##
Works on IOS

### CODE REVIEW

File: screens/JoinGroup.js [hunk 1]
```diff
@@ -46,6 +47,24 @@ class JoinGroup extends Component {
...
+    dateDict[today] = 0
+    db.child('/groups/'+groupID +'/groupMemberIds/'+userId).once("value")
+    .then(snapshot => {
+      if(!snapshot.val()) {
+        // create a userId reference to a list of dates
+        db.child('/groups/'+groupID+'/groupMemberIds/'+ userId).set(dateDict);
+      }
+    }).then(
+      this.props.navigation.navigate('Dashboard')
+    )
```
Comment (Peiyj) [inline screens/JoinGroup.js:65]: We only want to join the group if the groupId is valid. Else, we don't do anything in the backend. I think this block of code should be in the 
if(groupIds.includes(this.state.newGroupID)){
        .... <- here
        this.joinGroupSuccessfulAlert()
 }

**Review by Peiyj — COMMENTED** [review]
Almost there! Let's make that change and test it to see if it works! Make sure to pull from main as well.

**Comment by JipengSun** [comment]
> Almost there! Let's make that change and test it to see if it works! Make sure to pull from main as well.

Finish the group id precheck issue. I found bugs in completeDay buttons in our current main branch. However, I didn't fix it in join_group branch for the purpose-consistency. We can fix it tomorrow in another particular branch.

**Review by Peiyj — APPROVED** [review]
lgtm!

**Pull request merged**